package.json

```diff
@@ -127,7 +127,6 @@
 			"<rootDir>/test/unit/setup-wp-aliases.js"
 		],
 		"transform": {
-			"^.+\\.jsx?$": "babel-jest",
 			"\\.pegjs$": "<rootDir>/test/unit/pegjs-transform.js"
```

Comment:
The issue in `jest-config` is now resolved and we can merge options from preset with the local config. That's why we don't need to duplicate `"^.+\\.jsx?$": "babel-jest",` in here.